### ISSUE TITLE
About the edgeloss and softmax in KiUnet

### ISSUE THREAD

**Issue by slll12** [jeya-maria-jose/KiU-Net-pytorch#38]
Hi,
Thanks for your great work.
You used a threshold value of 0.5 in the code to handle the probability of the output, and whether such an operation can be derivable. In addition, there is processing of softmax in kiunet, the operations of these two parts makes me confused, can you explain it?
Best regards!

**Comment by zhouyizhuo** [comment]
Hi, I want to ask you which net did you used?
![image](https://github.com/jeya-maria-jose/KiU-Net-pytorch/assets/89564922/6bdfa4f3-8f3d-4eb5-9c95-8db973fbdc9c)
I tried to use kiunet_org, but it can't work
![image](https://github.com/jeya-maria-jose/KiU-Net-pytorch/assets/89564922/1351fcd5-0c1b-4945-83bf-2910993fc80b)
